content/english/run-node.md

```diff
@@ -38,19 +42,26 @@ version.
 
 ### Setting recommendations
 
+You can find community informations on [cosmos.directory/atomone](https://cosmos.directory/atomone).
```

Comment:
Community settings can be found at [cosmos.directory/atomone](https://cosmos.directory/atomone).